File: Brainrot.java
import java.util.*;
import java.io.*;

public class Brainrot {
    public static String shift(String s, int o) {
        StringBuilder res = new StringBuilder();
        for (char ch : s.toCharArray()) {
            if (!Character.isAlphabetic(ch) || !Character.isLowerCase(ch)) {
                return s;
            } else {
                char shiftedChar = (char) ((ch - 'a' + o) % 26 + 'a');
                res.append(shiftedChar);
            }
        }
        return res.toString();
    }

    public static ArrayList<String> repl(ArrayList<String> sl) {
        ArrayList<String> res = new ArrayList<>();
        for (String word : sl) {
            boolean w = false;
            for (int dist = 1; dist < 26; dist++) {
                String shifted = shift(word, dist);
                if (shifted.equals("skibidi") || shifted.equals("rizz") || shifted.equals("sigma") || shifted.equals("gyatt")) {
                    res.add(shifted);
                    w = true;
                    break;
                }
            }
            if (!w) {
                res.add(word);
            }
        }
        return res;
    }

    public static void main(String[] args) throws FileNotFoundException{
        Scanner sc = new Scanner(new File("Brainrot.in"));
        int n = sc.nextInt();
        sc.nextLine();
        while (n-- > 0) {
            ArrayList<String> l = new ArrayList<>(Arrays.asList(sc.nextLine().split(" ")));
            System.out.println(String.join(" ", repl(l)));
        }
    }
}

File: Brainrot.py
import sys
sys.stdin = open("Brainrot.in", "r")

def shift(s: str, o: int) -> str:
    res: str = ""
    for char in s:
        if not char.isalpha() or not char.islower():
            return s
        else:
            res += chr((ord(char) - ord('a') + o) % 26 + ord('a'))
    return res

def repl(sl: list[str]) -> list[str]:
    res = []
    for word in sl:
        w: bool = False
        for dist in range(1, 26):
            shifted: str = shift(word, dist)
            if shifted in ["skibidi", "rizz", "sigma", "gyatt"]:
                res.append(shifted)
                w = True
                break
        if not w:
            res.append(word)
    return res

n: int = int(input())

for i in range(n):
    l = input().split()
    print(" ".join(repl(l)))